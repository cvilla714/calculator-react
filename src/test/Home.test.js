import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('should display the correct content of the Home Page', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
