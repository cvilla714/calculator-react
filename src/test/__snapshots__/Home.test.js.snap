// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the correct content of the Home Page 1`] = `
<div
  className="container-fluid large"
>
  <h1
    className="ml-5 mt-5"
  >
    Welcome to our page!
  </h1>
  <p
    className="mx-5"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Quia ducimus, suscipit fuga facere adipisci nihil! Voluptatibus sunt veritatis ipsum voluptatum alias repellat qui aut, odio ad quibusdam dolor maxime illo. Nostrum, culpa amet nisi ea hic doloremque praesentium temporibus deserunt corporis commodi id voluptate doloribus voluptatem cumque. A quam porro hic architecto nesciunt voluptate, perferendis quibusdam dolor, cumque in ea! Sunt tempora voluptatum, neque facere voluptatem eum expedita error assumenda officiis eaque reprehenderit quam sit vero explicabo quibusdam esse, id est iusto optio laboriosam ad minima. Repellendus est placeat quis. Sed libero expedita similique exercitationem nemo, quidem aliquid sapiente id fugiat temporibus adipisci ipsum, veniam veritatis repudiandae delectus quis impedit consectetur. Quasi dolorum dolorem inventore facilis eius odio ab aliquam. Voluptate tempora, ducimus perferendis eius odio architecto animi dolor corrupti veniam nostrum amet, repellat consequatur modi quibusdam quidem illo. Quia velit incidunt nostrum sapiente nam culpa nesciunt error, doloribus ipsum! Voluptates quos fugit laborum tenetur? Vel porro iusto, voluptates facilis odio atque aperiam quo. Ducimus alias praesentium suscipit, error nihil nam provident perferendis iure quis cumque dolorum eveniet, dignissimos accusamus! Animi soluta corrupti nemo odio similique quaerat ratione voluptatum quas nisi excepturi ea laudantium consectetur dolorum tempore unde, quis provident reprehenderit explicabo? Possimus facere enim autem eius ut placeat obcaecati. Assumenda quaerat rem hic accusamus enim esse praesentium velit maiores ea iste nemo nesciunt, quam magnam repudiandae? Aliquid, veritatis. Autem molestiae sapiente possimus, at impedit quisquam sunt doloribus laborum beatae! Quis fugiat earum ipsa modi debitis. Non dolore, recusandae dolorum quaerat a earum corrupti pariatur cupiditate reiciendis ducimus ab sint dignissimos consequuntur! Culpa unde fugiat praesentium. Reiciendis aspernatur nostrum nisi. Dolorum neque fuga commodi vel asperiores eum quo. Minus dolor iusto officiis ad delectus quaerat vel a magnam exercitationem nobis, cumque hic? Recusandae, distinctio! Quidem itaque porro quos autem repellendus. Itaque exercitationem, necessitatibus consequuntur maxime illo blanditiis impedit earum vero quasi porro. Ullam repudiandae saepe nisi quo aliquam corporis eum aperiam, pariatur optio suscipit? Officiis, modi? Necessitatibus quis enim possimus. Doloribus amet sed sapiente, eum repudiandae facere quibusdam debitis unde totam aspernatur recusandae nobis pariatur sequi ex
  </p>
  <p
    className="mx-5"
  >
    harum repellat molestiae, sit veritatis? Facere aliquam esse, consectetur et consequatur nobis! Tenetur. Dolores veritatis deleniti ut, corporis repellendus esse repellat temporibus! Reiciendis quisquam, neque natus quidem sint velit accusantium nobis odit, dolore atque voluptas aliquid nihil necessitatibus laboriosam quia nemo, soluta placeat? Fuga sunt ipsum nisi recusandae ab? Expedita tempora placeat provident, doloribus itaque molestias asperiores enim unde ipsum rem, sapiente, fugiat inventore sint sequi ipsam exercitationem voluptate debitis cum veritatis nihil. Culpa inventore libero, recusandae architecto eius, consequatur quidem magnam totam voluptatibus cupiditate maxime voluptatum eos tempora quisquam est debitis similique, reprehenderit sunt at iusto quam pariatur cum nihil quia. Repellendus. Consequuntur quaerat tempora expedita debitis earum id deleniti sapiente eum ipsum architecto aperiam nemo eaque quidem, cupiditate repellendus. Quos mollitia fuga laboriosam laborum totam necessitatibus cupiditate quidem exercitationem. Eligendi, eaque. Commodi aperiam ducimus autem ex omnis non dolorum veniam, ipsa assumenda voluptatem ut quae molestias illo sed nam eos quibusdam labore sunt? Facere excepturi libero illum molestias corporis tenetur iste. Impedit aut neque adipisci quod, veritatis qui alias reiciendis eius dolore ipsa quasi nesciunt dolor officiis consequuntur molestiae dolorum voluptas recusandae doloremque quidem nulla cumque. Praesentium recusandae iusto vero alias! Minima recusandae enim ex, incidunt alias nihil quos voluptate dolor architecto. Et saepe neque nobis deleniti ab praesentium consectetur nulla aut voluptas laborum! Quod aliquam sed nam, saepe itaque minima? Accusamus totam quis corporis incidunt odio vero laudantium magni placeat. Amet aliquid totam beatae corporis qui delectus, id voluptates. Quidem accusantium quo quis, quibusdam dolores blanditiis laboriosam error cumque laudantium? Aliquam itaque temporibus ab at hic, explicabo sint maiores commodi molestias iste fuga eos incidunt neque laudantium cupiditate culpa et earum! Culpa voluptatibus officia architecto fugit nihil? Totam, corrupti explicabo? Eveniet corrupti totam non rem autem nihil ipsum corporis quaerat quas fuga officia aliquam, ad blanditiis perspiciatis, earum nostrum repudiandae laudantium. Dolorem dignissimos maxime tenetur vitae debitis fugit facere id. Perferendis, dolores, earum corrupti iste quis quo architecto distinctio explicabo ad quod mollitia ea magni soluta alias, vero accusamus enim. In molestias illo praesentium nulla eum, nisi fuga? Alias, nulla? Perspiciatis officiis ad ex. Libero totam unde nobis dolorum nulla suscipit beatae reiciendis necessitatibus voluptatem fugit, dignissimos aut illo perferendis officiis cumque officia sapiente nostrum accusamus, fuga eligendi laudantium. Perferendis? Beatae eos dolorum autem officiis recusandae provident
  </p>
</div>
`;
